File: output/kotlin/models/overridden_serializable/DisplayInformation.kt
package `overridden_serializable`

interface IDisplayInformation {
    val orientation: String
    val top: Int
    val left: Int
    val bottom: Int
    val right: Int
    val displayName: String
}

@Serializable
data class DisplayInformation(
    @SerialName("orientation") override val orientation: String,
    @SerialName("top") override val top: Int,
    @SerialName("left") override val left: Int,
    @SerialName("bottom") override val bottom: Int,
    @SerialName("right") override val right: Int,
    @SerialName("display_name") override val displayName: String,
) : IDisplayInformation

File: output/golang/server/models/DisplayInformation.go
package models

type DisplayInformation struct {
	Orientation string `json:"orientation" bson:"orientation"`
	Top int `json:"top" bson:"top"`
	Left int `json:"left" bson:"left"`
	Bottom int `json:"bottom" bson:"bottom"`
	Right int `json:"right" bson:"right"`
	DisplayName string `json:"display_name" bson:"display_name"`
}